index.html

```diff
@@ -173,6 +173,10 @@ <h3 id="build"> Build </h3>
       <td><img src='https://img.shields.io/sensiolabs/i/45afb680-d4e6-4e66-93ea-bcfa79eb8a87.svg' alt=''/></td>
       <td><code>https://img.shields.io/sensiolabs/i/45afb680-d4e6-4e66-93ea-bcfa79eb8a87.svg</code></td>
   </tr>
+  <tr><th> Gitter Chat: </th>
+    <td><img src='https://img.shields.io/gitter/gitterhq/irc-bridge.svg' alt=''/></td>
+    <td><code>https://img.shields.io/gitter/gitterhq/irc-bridge.svg</code></td>
+  </tr>
```

Comment:
The file `index.html` is auto-generated. Could you put this in `try.html` please?